src/epd_driver/include/epd_driver.h

```diff
@@ -460,8 +480,15 @@ EpdRect epd_difference_image_cropped(
  */
 EpdRect epd_difference_image(const uint8_t* to, const uint8_t* from, uint8_t* interlaced, bool* dirty_lines);
 
+/**
+ * Return the pixel color of a 4 bit image array
+ */
+uint8_t epd_get_pixel(int x, int y, int fb_width, int fb_height, const uint8_t *framebuffer);
```

Comment:
Please add a more detailed description of the parameters, like that the color is 0-255 as in epd_draw_pixels.

Reply:
Check in [last commit](https://github.com/martinberlin/epdiy-rotation/commit/1bbc2d477bf8d7447968f87218b394d401945459) if the detailed description is better suited